features/facebookImport/src/views/ministories/reportStory.jsx

```diff
@@ -25,7 +29,7 @@ class ReportStory extends Story {
     get active() {
         if (!this._neededReports) return true;
         for (const reportKey of this._neededReports) {
-            if (this.reports?.[reportKey] === undefined) return false;
+            if (this.reports?.[reportKey] === null) return false;
```

Comment:
Woah, scary :D I'd argue it can still be `undefined` - either if `this.reports` is not an object, or if `reportKey` is not a property on said object. If both is true and the explicit value happens to be `null`, it'd be `null` instead of course. If you know it can be `null`, I'm afraid you'll have to check for both :/